src/filesystem/impls/filer/lib/FileImport.js

```diff
@@ -10,12 +10,18 @@ define(function (require, exports, module) {
         FileSystemCache     = require("filesystem/impls/filer/FileSystemCache"),
         BrambleStartupState = brackets.getModule("bramble/StartupState");
 
+    // 1 MB
+    var MB = 1024 * 1024;
```

Comment:
do we want this in a utility class, so we can rely on `Sizes.KB`, `Sizes.MB`, etc?